.github/workflows/bump-sec-scanners-config-reusable.yml

```diff
@@ -87,29 +84,30 @@ jobs:
             echo "CREATE_PR=true" >> $GITHUB_ENV 
           fi
 
-      - name: Print Content of sec-scanners-config.yaml
         if: ${{ always() }}
+      - name: Print out sec-scanners-config.yaml
         shell: bash
         run: |
           FILE="sec-scanners-config.yaml"
           [ -f "${FILE}" ] && cat "${FILE}" || echo "${FILE} not found."
 
-      - name: Set Up Git
+      - name: Set up git
         if: ${{ env.CREATE_PR == 'true' }}
         env:
           GH_TOKEN: ${{ secrets.BOT_PAT }}
+          REPO: ${{ github.repository }}
         shell: bash
         run: |
           # set git username 
-          ghusername=$(curl -H "Authorization: token ${GH_TOKEN}" https://api.github.com/user)
+          ghusername=$(curl -s -H "Authorization: token ${GH_TOKEN}" https://api.github.com/user | jq '.login')
           git config user.name "${ghusername}"
           # set git mail address
-          ghmailaddress=$(curl -H "Authorization: token ${GH_TOKEN}" https://api.github.com/email)
+          ghmailaddress="${ghusername}@users.noreply.github.com"
```

Comment:
why do you want no reply email thing?

Reply:
because the `kyma-eventing-bot` will not answer emails. :)